### ISSUE TITLE
Support additional environment variables from secret

### ISSUE THREAD

**Issue by virtualdxs** [docker-mailserver/docker-mailserver-helm#109]
I don't want environment variables like RELAY_PASSWORD to be in my git repository, so I'd like to be able to configure a Secret where I can specify arbitrary environment variables to be added to the container via envFrom.

**Comment by cfis** [comment]
Sounds like a good idea to me - what to create a PR?

**Comment by virtualdxs** [comment]
Unfortunately, I'm not sure what the best way to do that would be, as after diving into it further I noticed that every environment variable is already specified in env and so envFrom would not be able to override it. I ended up manually configuring all the resources per the documentation instead of using this helm chart.

**Comment by cfis** [comment]
Yeah, probably should create a config map and secret and remove the envs from the deployment.

**Comment by emouawad** [comment]
+1 this is important for secret credentials like RELAY_PASSWORD
